highlight.io/components/QuickstartContent/traces/otlp.tsx

```diff
@@ -0,0 +1,29 @@
+import { QuickStartContent } from '../QuickstartContent'
+import { verifyTraces } from './shared-snippets'
+
+export const OTLPTracesContent: QuickStartContent = {
+	title: 'Tracing via the OpenTelemetry Protocol (OTLP)',
+	subtitle: `Learn how to export traces to highlight.io via one of the OpenTelemetry SDKs.`,
+	entries: [
+		{
+			title: 'Export your traces to the highlight.io collector.',
+			content:
+				'We host an OpenTelemetry collector endpoint at https://otel.highlight.io:4318/v1/traces. Configure your OpenTelemetry SDK to send traces via OTLP HTTPS to this endpoint. This will depend on which SDK you use in your app.',
```

Comment:
@Vadman97 - I wasn't sure about this - are there any special requirements for traces to have certain attributes set to be ingested correctly by our collector? Or is it enough for someone to just use our collector endpoint?

Reply:
the project_id attribute must be set on all traces. documented [here](https://www.highlight.io/docs/general/company/open-source/contributing/adding-an-sdk#recording-an-error) with a collector configuration [example here](https://www.highlight.io/docs/getting-started/backend-logging/file)